packetfabric/README.md

```diff
@@ -0,0 +1,45 @@
+# PacketFabric
+
+## Overview
+
+[PacketFabric][1] is a global Network as a Service (NaaS) provider that offers secure, private, and on-demand connectivity services. 
+
+You can use PacketFabric to quickly and easily build a robust network between cloud platforms, SaaS providers, and hundreds of colocation centers around the [world][2].
+
+With this integration, you can leverage Datadog to monitor your PacketFabric network traffic data.
```

Comment:
Can you add what traffic data you are monitoring to datadog and why this is helpful?

Reply:
@solababs Could you please update the above to specify what network traffic data you are monitoring and the value? thanks!